### PR TITLE
Added New Stops

### PR DESCRIPTION
**Pull request by steets250**
Completed stop list for the gold line.

### CODE REVIEW

File: data/lax_tap/mapping.csv [hunk 1]
```diff
@@ -6,7 +6,6 @@ reader_id,stop_code,stop_id
 14,80205S,
 15,80201S,
 16,80212S,
-17,80214S,
```
Comment (micolous) [inline data/lax_tap/mapping.csv]: 17 was incorrect?

Comment (steets250) [inline data/lax_tap/mapping.csv]: Stops on the Red Line go in order, from 11 to 25, and the pattern seemed to indicate that Union Station is stop 17, but when I actually went, it had the ID of 7, so I removed that ID from the list.

Comment (steets250) [inline data/lax_tap/mapping.csv]: So yes, 17 is incorrect.

**Comment by micolous** [comment]
Thanks for confirming. I'll merge this now.

**Comment by micolous** [comment]
To-do for me is to update the GTFS and then rebuild the sqlite3 database.

**Pull request merged**